docs/history-tracking.md

```diff
@@ -18,24 +23,26 @@ In our case, history tracking is part of a slightly larger endeavour labelled "v
 control", in which updates to records in the database are expected from time to time, 
 and we need to track such updates. 
 
-The canonical use case is as follows: A researcher downloads data from our database,
+The canonical use case is as follows: A researcher downloads data from the CRMP database,
```

Comment:
crmp vs metnorth?

Might be worth adding to the basic terminology, crmp vs pycds vs metnorth